bin/docker-build

```diff
@@ -9,6 +9,9 @@ fi
 
 bin/docker-build-controller
 bin/docker-build-web
-bin/docker-build-proxy
 bin/docker-build-proxy-init
 bin/docker-build-cli
+
+bin/docker-build-proxy \
+    --build-arg="SKIP_TESTS=${SKIP_TESTS:-}" \
+    --build-arg="BUILD_UNOPTIMIZED=${PROXY_UNOPTIMIZED:-}"
```

Comment:
can we settle on one of `BUILD_UNOPTIMIZED` / `PROXY_UNOPTIMIZED`. i'd prefer the latter as it more explicitly affects the proxy build.

Reply:
in the context of proxy/Dockerfile, it's weird to have PROXY_UNOPTIMIZED, as it relates to much more than just the proxy.  In the context of bin/docker-build, BUILD_UNOPTIMIZED is too generic. I suppose we can just use PROXY_UNOPTIMIZED but i think it's weird...

Reply:
I agree with siggy.